plutus-ledger-api/test-plugin/Spec/golden/9.2/Short.stat.golden

```diff
@@ -1,21 +1,21 @@
-({cpu: 5819100 | mem: 25400})
-({cpu: 10672100 | mem: 46500})
-({cpu: 16625882 | mem: 69804})
-({cpu: 20457371 | mem: 85206})
-({cpu: 23835153 | mem: 97310})
-({cpu: 31603198 | mem: 128414})
-({cpu: 44205186 | mem: 176624})
-({cpu: 43792795 | mem: 176122})
-({cpu: 51542186 | mem: 208524})
-({cpu: 59371679 | mem: 237032})
-({cpu: 83158026 | mem: 328650})
-({cpu: 77267223 | mem: 306844})
-({cpu: 100929378 | mem: 399060})
-({cpu: 115575158 | mem: 455670})
-({cpu: 146823868 | mem: 571500})
-({cpu: 156084569 | mem: 606208})
-({cpu: 198957295 | mem: 766946})
-({cpu: 204775120 | mem: 790948})
-({cpu: 236346454 | mem: 914568})
-({cpu: 390817706 | mem: 1513172})
-({cpu: 927859238 | mem: 3396606})
\ No newline at end of file
+({cpu: 4439100 | mem: 19400})
+({cpu: 6164100 | mem: 26900})
+({cpu: 8715491 | mem: 36702})
+({cpu: 10430980 | mem: 42904})
```

Comment:
These tests are a bit better, but still suffer from the same problem. But even then, at least some of the numbers we see here _are_ representative of the actual cost changes.